packages/common/src/types.ts

```diff
@@ -456,9 +468,10 @@ export interface IScheduleCourse {
 export type SupportedConcentrations = {
   concentrations: string[];
   minRequiredConcentrations: number;
+  verified: boolean;
```

Comment:
This might be just me not being familiar with Graduate codebase after a while, but why is the `verified` property here necessary for concentration when a major already indicates that? (If a major is `verified`, that should mean the whole major, including its concentration, is verified right?)

Reply:
The list of majors endpoint for the add plan and edit plan modals use some kind of Record mapping relevant major information. Since we want to mark majors with a [BETA] tag in the modal, we also want to pass along the flag from the metadata if the major is verified. I guess technically the name of this type can be updated to SupportedMajorInformation, including concentrations and now this verified tag.